content/concept.md

```diff
@@ -65,12 +65,12 @@ allows us to visualize different HTTP interfaces for Linked Data _together_.
 **SPARQL endpoints** are easy for clients,
 as they allow highly specific fragment selection.
 However, they also have the highest server cost
-which makes it expensive to host them with [decent availability](http://sw.deri.org/~aidanh/docs/epmonitorISWC.pdf).
+which makes it expensive to host them with [decent availability](http://aidanhogan.com/docs/epmonitorISWC.pdf).
 If you don't want to depend on such an endpoint,
 you download a **data dump**,
 but then you're querying a local source instead of the Web.
 With **subject pages**, servers do minimal effort,
-but clients need to [work hard](http://squin.sourceforge.net/) to solve simple queries.
+but clients need to [work hard](https://squin.sourceforge.net/) to solve simple queries.
```

Comment:
This one is only available on http apparently.

Reply:
Oops, sorry